awx/ui/client/src/notifications/notification-templates-list/list.controller.js

```diff
@@ -152,9 +152,14 @@
                                      content: `<i class="fa fa-check-circle Toast-successIcon"></i> <b>${name}:</b> Notification sent.`
                                  });
                                  $state.reload();
-                             } else if (res && res.data && res.data.status && res.data.status === "failed") {
+                             } else if (res && res.data && res.data.status && res.data.status === "failed" && res.data.error === "timed out") {
                                  ngToast.danger({
-                                     content: `<i class="fa fa-exclamation-triangle Toast-successIcon"></i> <b>${name}:</b> Notification failed.`
+                                     content: `<div><i class="fa fa-exclamation-triangle Toast-successIcon"></i> <b>${name}:</b> ${i18n._("Notification timed out.")}</div>`
+                                 });
+                                 $state.reload();
+                             } else if (res && res.data && res.data.status && res.data.status === "failed" && res.data.error !== "[Errno 115] Operation now in progress") {
```

Comment:
Don't think we need this block anymore.

Reply:
Ah, sorry, I mean we don't need the `res.data.error !== "[Errno 115] Operation now in progress"` check.

Reply:
Removed as requested, tested it and everything still works fine so ✔️